configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
 The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on a UI.
 
 openHAB persists Item states in a database, and most popular databases are supported.
-You may have more that one persistence add-on loaded, and each of these may be configured independently.
+You may have more than one persistence add-on loaded, and each of these may be configured independently.
 
 A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
 
-## Persistence Add-on Configuration
+## Add-on Configuration
```

Comment:
Once about the "Configuration" headline issue. I think it would best be solved if you simply removed this one.
wdyt?

Reply:
Yes - that makes sense.